src/Microsoft.OData.Core/ODataContextUrlInfo.cs

```diff
@@ -403,8 +403,9 @@ private static string CreateSelectExpandContextUriSegment(SelectExpandClause sel
         /// <returns>The generated expand string.</returns>
         private static string ProcessSubExpand(string expandNode, string subExpand, ODataVersion version)
         {
-            return string.IsNullOrEmpty(subExpand) && version <= ODataVersion.V4 ? null :

```

Comment:
>string.IsNullOrEmpty(subExpand) [](start = 19, length = 31)

Should we keep "string.IsNullOrEmpty(subExpand) " check and return "null"?

Reply:
We do need to include the case of null and return "()" for expanded entity.

---
In reply to: [224171315](https://github.com/OData/odata.net/pull/1286#discussion_r224171315) [](ancestors = 224171315)